multi-node/aws/pkg/config/config.go

```diff
@@ -108,6 +118,12 @@ type Cluster struct {
 	RecordSetTTL             int               `yaml:"recordSetTTL"`
 	HostedZone               string            `yaml:"hostedZone"`
 	StackTags                map[string]string `yaml:"stackTags"`
+	Subnets                  []Subnet          `yaml:"subnets"`
```

Comment:
This field (and it's subfields) need to be added to `pkg/config/templates/cluster.yaml`